cves/CVE-2010-4476.yml

```diff
@@ -99,8 +108,10 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
+  answer: |
+    There is no evidence in the Tomcat code that unit tests for these
+    files existed.
+  code: false
   fix:
```

Comment:
Based on your above description, "fix" should be false

Reply:
Updated all that you mentioned, thanks!